package.nls.de.json

```diff
@@ -19,6 +19,7 @@
   "configuration.presets.jsonOfficial.description": "Wenn auf 'true' gesetzt, wird die Erweiterung das offizielle JSON Icon benutzen.",
   "configuration.presets.hideFolders.description": "Wenn auf 'true' gesetzt, werden keine Ordner-Icons angezeigt.",
   "configuration.presets.foldersAllDefaultIcon.description": "Wenn auf 'true' gesetzt, werden keine speziellen Ordner-Icons angezeigt.",
+  "configuration.customIconFolderPath.description": "",
```

Comment:
`Der absolute Pfad zu dem Oberverzeichnis wo die benutzerdefinierten Icons in einem Unterordner liegen.`